src/cloud-resources/src/crd/materialize.rs

```diff
@@ -77,8 +78,8 @@ pub mod v1alpha1 {
     }
 
     impl Materialize {
-        pub fn cockroach_secret_name() -> String {
-            "cockroach-password".to_string()
+        pub fn backend_secret_name(&self) -> String {
```

Comment:
nit: feels like this should have `metadata` in the name

Reply:
it's not just metadata though, we also configure the persist blob store here